diff --git a/dist/index.mjs b/dist/index.mjs
index 1720bad9ba3d875f95cdfcba4e3190a44d7a1145..56fe13a36fec5346c5395f5f7005d035380de8c8 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -7,10 +7,20 @@ import { createTwoslasher } from "twoslash";
 * Factory function to create a Shiki transformer for twoslash integrations.
 */
 function transformerTwoslash(options = {}) {
-	return createTransformerFactory(createTwoslasher({
-		cache: options?.cache,
-		compilerOptions: { moduleResolution: 100 }
-	}), rendererRich(options.rendererRich))(options);
+	const twoslashOptions = {
+    cache: options.cache,
+    compilerOptions: {
+      moduleResolution: 100,
+    },
+  };
+
+	const tsModule = options.tsModule || options.twoslashOptions?.tsModule;
+
+ 	if (tsModule) {
+    twoslashOptions.tsModule = tsModule;
+  }
+
+	return createTransformerFactory(createTwoslasher(twoslashOptions), rendererRich(options.rendererRich))(options);
 }
 
 //#endregion
