diff --git a/dist/index.mjs b/dist/index.mjs
index c18069c87a507d2fc323d4b633f3941594c9cd28..8c7a81fcaf374d4ddd8527eee6e04eec5a5e74bc 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -9,7 +9,8 @@ function transformerTwoslash(options = {}) {
       cache: options?.cache,
       compilerOptions: {
         moduleResolution: 100
-      }
+      },
+      tsModule: options?.twoslashOptions?.tsModule,
     }),
     rendererRich(options.rendererRich)
   )(options);
